server/apps/postgres-app/src/main/java/org/apache/james/PostgresJamesServerMain.java

```diff
@@ -94,20 +97,26 @@ public class PostgresJamesServerMain implements JamesServerMain {
         new ActiveMQQueueModule(),
         new BlobExportMechanismModule(),
         new PostgresDelegationStoreModule(),
-        new DefaultProcessorsConfigurationProviderModule(),
         new PostgresMailboxModule(),
         new PostgresDeadLetterModule(),
         new PostgresDataModule(),
         new MailboxModule(),
-        new NoJwtModule(),
-        new RawPostDequeueDecoratorModule(),
         new SievePostgresRepositoryModules(),
         new TaskManagerModule(),
         new MemoryEventStoreModule(),
         new TikaMailboxModule());
 
+    public static final Module JMAP = Modules.combine(
+        new PostgresJmapModule(),
+        new JmapEventBusModule(),
+        new PostgresDataJmapModule(),
+        new MemoryPushSubscriptionModule(),
+        new JMAPServerModule(),
+        new JmapTasksModule(),
+        new JMAPListenerModule());
```

Comment:
What about the other `JMAPEventBusModule` ?

Reply:
added